src/style/dark.scss

```diff
@@ -0,0 +1,150 @@
+// Darkly 4.5.2
+// Bootswatch
```

Comment:
Although we used that file as template, it probably has not too much in common with the result. Could this reference therefore be deleted?

Reply:
done